static/usage/card/basic/demo.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Card</title>
+  <link rel="stylesheet" href="../../common.css" />
+  <script src="../../common.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@6/dist/ionic/ionic.esm.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@6/css/ionic.bundle.css" />
+
+  <style>
+    .container {
+      max-width: 400px;
+      margin: 0 auto;
+    }
+
+    /* iOS places the subtitle above the title */
+    ion-card-header.ios {
+      display: flex;
+      flex-flow: column-reverse;
+    }
```

Comment:
I know Ben requested this, but this seems wrong. If the default iOS layout in Ionic is incorrect, shouldn't we fix that internally instead?

Reply:
I thought about this too, but I was wondering if it would be too "magical" if we automatically switched them on users. Should we open discussion for it?

Reply:
I opened a pull request for implementing this in the framework: https://github.com/ionic-team/ionic-framework/pull/26084

The alternative is we continue to do as we did in past examples and switch the subtitle & title so it looks right in iOS and wrong in Material Design.